perllib/FixMyStreet/Cobrand/Zurich.pm

```diff
@@ -1435,7 +1435,7 @@ sub report_new_munge_before_insert {
 sub report_on_private_contacts {
     my ($self, $category) = @_;
 
-    return [ FixMyStreet::DB->resultset('Contact')->search({category => $category})->all ];
+    return [ FixMyStreet::DB->resultset('Contact')->search({state => 'inactive', category => $category})->all ];
```

Comment:
This works okay, but as I suggested I think `FixMyStreet::DB->resultset('Contact')->not_deleted->search({ category => $category })->all` is using the shared filter then, that other code like this uses? Unless there was a reason not to use that you didn't say.

Reply:
Sort of misunderstood what you said in the previous comment was all.